### PR TITLE
updated-user-test-data

### PR DESCRIPTION
**Pull request by alinolteanufr**
Added new user data for testing accounts flow for v4

### CODE REVIEW

File: secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataCreator.java [hunk 1]
```diff
@@ -71,16 +71,8 @@
 import com.forgerock.sapi.gateway.rs.resource.store.repo.mongo.customerinfo.FRCustomerInfoRepository;
 
 import lombok.NoArgsConstructor;
-import uk.org.openbanking.datamodel.v3.account.OBBeneficiary5;
-import uk.org.openbanking.datamodel.v3.account.OBReadBalance1DataBalanceInner;
-import uk.org.openbanking.datamodel.v3.account.OBReadDirectDebit2DataDirectDebitInner;
-import uk.org.openbanking.datamodel.v3.account.OBReadOffer1DataOfferInner;
+import uk.org.openbanking.datamodel.v4.account.*;
```
Comment (wmorrison365fr) [inline secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataCreator.java:74]: Can you reference the exact classes please.

File: secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataExporter.java [hunk 2]
```diff
@@ -56,16 +58,7 @@
...
-import uk.org.openbanking.datamodel.v3.account.OBParty2;
-import uk.org.openbanking.datamodel.v3.account.OBReadBalance1DataBalanceInner;
-import uk.org.openbanking.datamodel.v3.account.OBReadDirectDebit2DataDirectDebitInner;
-import uk.org.openbanking.datamodel.v3.account.OBReadOffer1DataOfferInner;
-import uk.org.openbanking.datamodel.v3.account.OBReadProduct2DataProductInner;
-import uk.org.openbanking.datamodel.v3.account.OBScheduledPayment3;
-import uk.org.openbanking.datamodel.v3.account.OBStandingOrder6;
-import uk.org.openbanking.datamodel.v3.account.OBStatement2;
-import uk.org.openbanking.datamodel.v3.account.OBTransaction6;
+import uk.org.openbanking.datamodel.v4.account.*;
```
Comment (wmorrison365fr) [inline secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataExporter.java:61]: Likewise.

File: secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataUpdater.java [hunk 3]
```diff
@@ -42,20 +42,21 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 import org.springframework.web.server.ResponseStatusException;
-import uk.org.openbanking.datamodel.v3.account.*;
+import uk.org.openbanking.datamodel.v4.account.*;
```
Comment (wmorrison365fr) [inline secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataUpdater.java:45]: And again (imported classes).

File: secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-datamodel/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/datamodel/account/FRAccountData.java [hunk 4]
```diff
@@ -19,8 +19,8 @@
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-import uk.org.openbanking.datamodel.v3.account.*;
-
+import uk.org.openbanking.datamodel.v4.account.*;
```
Comment (wmorrison365fr) [inline secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-datamodel/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/datamodel/account/FRAccountData.java:22]: Likewise :-) 
(Again, realise it was originally like this)

**Review by wmorrison365fr — APPROVED** [review]

File: secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataCreator.java (hunk 1 above)
Comment (alinolteanufr) [reply secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataCreator.java]: yep, sorry about this, Intellij has this behavior when I import 5 or more classes from the same package.

**Review by wmorrison365fr — APPROVED** [review]
Thanks Alin.

**Review by wmorrison365fr — APPROVED** [review]

**Pull request closed without merging**